### PR TITLE
Fix bronze dragon breath weapons

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: packs/data/pathfinder-bestiary.db/adult-bronze-dragon-spellcaster.json [hunk 1]
```diff
@@ -3391,7 +3391,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:33|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:33] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary.db/adult-bronze-dragon-spellcaster.json:3394]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|dc:33|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:33|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
We want these parameters in the order listed in the style guide for consistency.

File: packs/data/pathfinder-bestiary.db/young-bronze-dragon-spellcaster.json [hunk 2]
```diff
@@ -2526,7 +2526,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:60]{60-foot line} that deals [[/r 6d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:28|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:60]{60-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:28] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary.db/young-bronze-dragon-spellcaster.json:2529]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:60]{60-foot line} that deals [[/r 6d12[electricity]]] damage (@Check[type:reflex|dc:28|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:60]{60-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:28|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

File: packs/data/pathfinder-bestiary.db/ancient-bronze-dragon.json [hunk 3]
```diff
@@ -1085,7 +1085,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:100]{100-foot line} that deals [[/r 12d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:40|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:100]{100-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:40] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary.db/ancient-bronze-dragon.json:1088]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:100]{100-foot line} that deals [[/r 12d12[electricity]]] damage (@Check[type:reflex|dc:40|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:100]{100-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:40|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

File: packs/data/pathfinder-bestiary.db/adult-bronze-dragon.json [hunk 4]
```diff
@@ -886,7 +886,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:33|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:33] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary.db/adult-bronze-dragon.json:889]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|dc:33|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:33|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

File: packs/data/pathfinder-bestiary.db/ancient-bronze-dragon-spellcaster.json [hunk 5]
```diff
@@ -4245,7 +4245,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:100]{100-foot line} that deals [[/r 12d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:40|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:100]{100-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:40] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary.db/ancient-bronze-dragon-spellcaster.json:4248]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:100]{100-foot line} that deals [[/r 12d12[electricity]]] damage (@Check[type:reflex|dc:40|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:100]{100-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:40|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

File: packs/data/pathfinder-bestiary.db/young-bronze-dragon.json [hunk 6]
```diff
@@ -667,7 +667,7 @@
...
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:60]{60-foot line} that deals [[/r 6d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:28|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:60]{60-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:28] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary.db/young-bronze-dragon.json:670]:
```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:60]{60-foot line} that deals [[/r 6d12[electricity]]] damage (@Check[type:reflex|dc:28|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:60]{60-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:28|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

**Pull request merged**